.appveyor.yml

```diff
@@ -0,0 +1,6 @@
+version: 3.7.0a0.{build}
+clone_depth: 5
+build_script:
```

Comment:
Unless we can mandate that setting for all checkouts on Windows (by checking something in, not just devguide fiat), I think we should fix our stuff such that the build succeeds and the tests pass regardless of the core.autocrlf setting.  Otherwise we'll always have people popping up with the wrong core.autocrlf setting and test failures.